src/org/javarosa/core/model/FormDef.java

```diff
@@ -973,15 +973,16 @@ public String fillTemplateString(String template, TreeReference contextRef,
     }
 
     /**
-     * Identifies the itemset in the backend model, and creates a set of
-     * SelectChoice objects at the current question reference based on the data
+     * Identifies the `itemset` in the backend model, and creates a set of
```

Comment:
Markdown here and in the rest of the comment!

Reply:
@lognaturel Eep! Let me take care of these.